code/app/src/main/res/layout/activity_facility_profile.xml

```diff
@@ -0,0 +1,53 @@
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:padding="16dp">
+
+    <LinearLayout
+        android:layout_width="match_parent"
```

Comment:
I think this functionality is already implemented, please double check with James